codice/ddf
# Address some SonarQube findings and missing unit tests in catalog-solr-offline-gazetteer

#### What does this PR do?
Addresses SonarQube findings in BuildGazetteerSuggesterIndexCommand
https://sonarcloud.io/project/issues?id=ddf&issues=AXPkHocw60FsyeUshtpL&open=AXPkHocw60FsyeUshtpL
![image](https://user-images.githubusercontent.com/8041246/94489957-4fc78800-019a-11eb-926b-8900c0b939bd.png)
![image](https://user-images.githubusercontent.com/8041246/94490090-9ae19b00-019a-11eb-9de0-98e463dd04a0.png)
![image](https://user-images.githubusercontent.com/8041246/94596577-2ca3e380-0241-11eb-8465-5d707c1778a4.png)

#### Who is reviewing it? 
@rzwiefel @blen-desta @garrettfreibott @SmithJosh @bakejeyner @stustison 

#### How should this be tested?
Confirm that the `offline-solr-gazetteer` commands still work
- see the `Standalone Solr Offline Gazetteer Feature` section in the docs
- see https://github.com/codice/ddf/pull/6153

#### Checklist:
- [n/a] Documentation Updated
- [n/a] Update / Add Threat Dragon models
- [n/a] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.
